### DISCUSSION TITLE
有没有人被硅谷裁员的？

### DISCUSSION THREAD

**Discussion by leon-fudan** [The-Run-Philosophy-Organization/run#609] [General]
硅谷最近大裁员，不知道有没有人run 过去被裁到的？人家还给不给你工作签了？失业是不是也可以有地方住（街头帐篷不算）。
不过我估摸着群里run的这些人也不一定能进硅谷大公司，还是比较安全的。祝福大家。

**Comment by sseed5566** [comment]
群里的润人进得了硅谷也不是不可能，硅谷有很多岗位是契合润人的，比如餐饮店，搬运工，厕所清洁工，垃圾工，外卖，杀虫工作，装修苦力，试药的。。。等等
